spec/services/reports/sp_active_users_report_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+describe Reports::SpActiveUsersReport do
+  subject { described_class.new }
+
+  it 'is empty' do
+    expect(subject.call).to eq('[]')
```

Comment:
is our convention that we at least add one row to make sure the report isn't completely empty?